InnerEye/ML/utils/dataset_util.py

```diff
@@ -154,8 +154,8 @@ def store_and_upload_example(dataset_example: DatasetExample,
         folder = images_folder
     else:
         folder = args.example_images_folder if args else Path("")
-    if folder != "" and not os.path.exists(folder):
-        os.mkdir(folder)
+    if folder != "":
```

Comment:
I'm slightly suspicious of this != "" check. Is that supposed to catch the Path("") case in the previous line? I doubt that it would, because you would end up comparing a string and a Path. 
Maybe simpler to use `folder = (args.example_folder if args else images_folder) or Path.cwd()`, then you don't need to guard the makedirs call at all.